test/integration/model/findOne.test.js

```diff
@@ -924,9 +930,13 @@ describe(Support.getTestDialectTeaser('Model'), () => {
       });
 
       it('throws error when record not found by findByPk', function() {
+        // Postgres complains about invalid integer
+        const rejectArgs = dialect === 'postgres' ? 
+          [Sequelize.DatabaseError, 'invalid input syntax for integer: "4.732322332323333e+30"'] : [Sequelize.EmptyResultError];
```

Comment:
This is kind of unusual, usually you want `.catch` always be called with an error object, not a tuple and instead store error metadata on the error object itself.

Reply:
It was there before. Perhaps I moved it around, and git attributes this change to me.